services/datamanager/builtin/cmd/main.go

```diff
@@ -0,0 +1,100 @@
+// package main prints a disk summary of the builtin data manager's capture directory
```

Comment:
When does this run? Is it on demand by a viam engineer or SE or on a schedule?

Reply:
It is never run unless you (a viam dev, an SE engineer, etc) compile it and run it as a command line utility. j

It is purely there as a convenience.

no viam software will run it automatically